qt/scientific_interfaces/Inelastic/Manipulation/InelasticDataManipulation.cpp

```diff
@@ -63,7 +63,7 @@ void InelasticDataManipulation::initLayout() {
   addMVPTab<InelasticDataManipulationSqwTab, InelasticDataManipulationSqwTabView>("S(Q, w)");
   addMVPTab<InelasticDataManipulationMomentsTab, InelasticDataManipulationMomentsTabView>("Moments");
   addMVPTab<InelasticDataManipulationElwinTab, InelasticDataManipulationElwinTabView>("Elwin");
-  addTab<InelasticDataManipulationIqtTab>("Iqt");
+  addMVPTab<InelasticDataManipulationIqtTab, InelasticDataManipulationIqtTabView>("Iqt");
```

Comment:
Now that all the tabs are added using `addMVPTab`, can we remove the `addTab` function? Also maybe we should rename `addMVPTab` to `addTab`?